src/utilities/request.jl

```diff
@@ -166,8 +184,13 @@ function submit_request(aws::AbstractAWSConfig, request::Request; return_headers
                 ),
             )
         end
+        return false
     end
 
+    delays = AWSExponentialBackoff(; max_attempts=3)
+
+    retry(upgrade_error(get_response); check=check, delays=delays)()
```

Comment:
Requires that `() ->` is removed from `upgrade_error`
```suggestion
    retry(upgrade_error ∘ get_response; check=check, delays=delays)()
```

Reply:
This doesn't work, since `upgrade_error` now calls its input as a function. So we do `result = get_response()`, then `ugrade_error(result)`. I think we really need the function-wrapper approach we had before to do this.

Reply:
You are correct. I seem to be having an off day